scripts/extract-l10n.sh

```diff
@@ -8,10 +8,10 @@
 #   - Update your code
 #   - Extract new strings and push to the .po files
 #
-# If you're running the script manually please make sure to expose
+# For the fully automated experience make sure to expose
 # the following variables to the environment:
 #   - GITHUB_TOKEN (to the github token of addons-robot,
-#                   talk to tofumatt or cgrebs)
+#                   talk to @diox or @muffinresearch)
 #   - TRAVIS_REPO_SLUG="mozilla/addons-server"
 #   - TRAVIS_BRANCH="master"
```

Comment:
This doesn't appear to be used at all (despite the erroring if it's not set).

Reply:
This is purely if used in travis to make sure it's not running against branches other than master

Reply:
okay, that makes sense.